tests/test_create_conference.py

```diff
@@ -0,0 +1,339 @@
+            for profile_json in data['profiles']:
+                if not client.search_profiles(ids=[profile_json['id']]):
+                    # If the profile hasn't already been posted get the data and create the user
+                    new_id = profile_json['id']
+                    email = profile_json.get('content').get('preferredEmail') or profile_json.get('content').get('emails')[0]
+                    first_name = profile_json['id'][1:-1].split('_')[0]
+                    last_name = profile_json['id'][1:-1].split('_')[-1]
+                    helpers.create_user(email, first_name, last_name)
+                    
+                    # A pre-generated user may not have 1 at the end
+                    # In this case, add the same name to their profile, create a group for this new ID and rename the profile
+                    curr_id = client.get_profile(email).id                   
+                    if new_id != curr_id:
+                        client.post_profile(openreview.Profile(
+                            referent = curr_id, 
+                            invitation = '~/-/invitation',
+                            signatures = ['openreview.net'],
+                            content = {},
+                            metaContent = {
+                                'names': { 
+                                    'values': [{ 
+                                        'first': first_name,
+                                        'middle': '',
+                                        'last': last_name,
+                                        'username': new_id }],
+                                    'weights': [1] }
+                            }))
+                        client.post_group(
+                            openreview.Group(
+                                id = new_id,
+                                readers = [new_id],
+                                writers = ['openreview.net'],
+                                signatories = [new_id],
+                                signatures = ['openreview.net']
+                            )
+                        )
+
+                        client.rename_profile(curr_id, new_id)
+
+                    assert client.get_profile(new_id)
+                    assert client.get_profile(new_id).id == new_id
+        
+        # Post a small number of reviewers to the ABC.cc group used for testing the expertise model
+        # to reduce test time
+        with open('tests/data/expertiseServiceData.json') as json_file:
+            data = json.load(json_file)
+        post_profiles(data)
+        members = data['groups']['ABC.cc']['members']
+
+        group = openreview.Group(
+            id = 'ABC.cc',
+            readers = ['everyone'],
+            writers = ['openreview.net'],
+            signatories = ['openreview.net'],
+            signatures = ['openreview.net'],
+            members = members
+        )
+        client.post_group(group)
```

Comment:
if you are creating a conference, why don't you use `conference.get_reviewers_id()` and add the members to that group. It is a more realistic test.